README.md

```diff
@@ -17,9 +19,7 @@ However, the intended use-case for **WebAuthn-Local-Client** is to allow [Local-
 
 **Note:** This package *may* be used in combination with a traditional FIDO2 server application architecture, but does not include any specific functionality for that purpose. For server integration with `WebAuthn`, you may instead consider alternative libraries, like [this one](https://github.com/passwordless-id/webauthn) or [this one](https://github.com/raohwork/webauthn-client).
 
-## Demo App
-
-A demo app (using this library) that showcases usage of WebAuthn and also cryptography to locally encrypt/decrypt user data can be found [here](https://vella.ai/auth/).
+## Demo Video
```

Comment:
Is there a reason not to move the demo **VIDEO** link up next to (or below) the link to the same demo as the video itself demonstrates? That seems like the best place for that, rather than separated by several paragraphs.

Reply:
also, this link is to a temporary asset upload (here on github), and is not a suitable place to link to for the video. we need the video posted to something like youtube / vimeo /etc.